limitador/src/storage/distributed/mod.rs

```diff
@@ -356,29 +268,47 @@ impl CrInMemoryStorage {
 struct CounterKey {
     namespace: Namespace,
     seconds: u64,
+    max_value: u64,
```

Comment:
Should we push the max_value into the value portion of the map entry?

Reply:
maybe a changed_at is not needed, as that's something that's a limits configuration value, so that change should be rolled to all peers at the same time.